From 73973e7a4df5d34bb107e54a2c29908c4a51d33b Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 23:11:11 -0400
Subject: [ethernet] cxgb4vf: Adds device ID for few more Chelsio T4 Adapters

Message-id: <1410477072-95160-7-git-send-email-svemuri@redhat.com>
Patchwork-id: 93903
O-Subject: [RHEL7.1 PATCH BZ 1070945 6/7] cxgb4vf: Adds device ID for few more Chelsio T4 Adapters
Bugzilla: 1070945
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit dde3aadf53ce4d5c5857d87619eeb9ff777a9c2f)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
index 5285928..b25320a 100644
--- a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
@@ -2924,6 +2924,15 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4vf_pci_tbl) = {
  CH_DEVICE(0x480a, 0),   /* T404-bt */
  CH_DEVICE(0x480d, 0),   /* T480-cr */
  CH_DEVICE(0x480e, 0),   /* T440-lp-cr */
+ CH_DEVICE(0x4880, 0),
+ CH_DEVICE(0x4880, 1),
+ CH_DEVICE(0x4880, 2),
+ CH_DEVICE(0x4880, 3),
+ CH_DEVICE(0x4880, 4),
+ CH_DEVICE(0x4880, 5),
+ CH_DEVICE(0x4880, 6),
+ CH_DEVICE(0x4880, 7),
+ CH_DEVICE(0x4880, 8),
  CH_DEVICE(0x5800, 0), /* T580-dbg */
  CH_DEVICE(0x5801, 0), /* T520-cr */
  CH_DEVICE(0x5802, 0), /* T522-cr */
-- 
1.7.1